cmd/undo-main.go

```diff
@@ -49,6 +49,10 @@ var undoFlags = []cli.Flag{
 		Name:  "dry-run",
 		Usage: "fake an undo operation",
 	},
+	cli.StringFlag{
+		Name:  "action",
```

Comment:
Unfortunately --action and --last doesn't work well together, maybe we should require last=1 if --action is specified.

Reply:
Ok